feedback/templates/manage/_filter_field.html

```diff
@@ -0,0 +1,19 @@
+<div class="fieldWrapper feedback-filter">
+	{{ field.errors }}
+	{% if not hide_label %}
+		<label for="{{ field.id_for_label }}"
+			{% if label_classes %}
+				class="{{ label_classes }}"
+			{% endif %}
+            {% if field.help_text %}
```

Comment:
Indentation looks strange here. Maybe mixed tabs and spaces?

Reply:
Oops, good point. Indeed spaces instead of tabs slipped in to that row when cleaning git history.